tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun.go

```diff
@@ -112,16 +117,28 @@ func init() {
 	if err != nil {
 		log.Fatalf("failed to marshal patch bytes in order to cancel: %v", err)
 	}
+	params = db.ConnectionParams{}
+	params.LoadMySQLDefaults()
+}
+
+func isCachingEnabled(run *v1alpha1.Run) bool {
+	return run.Labels["pipelines.kubeflow.org/cache_enabled"] == "true"
 }
 
 // ReconcileKind compares the actual state with the desired, and attempts to converge the two.
 // It then updates the Status block of the Run resource with the current status of the resource.
 func (c *Reconciler) ReconcileKind(ctx context.Context, run *v1alpha1.Run) pkgreconciler.Event {
 	var merr error
 	logger := logging.FromContext(ctx)
+	err := CacheStore.Connect(params)
```

Comment:
Can we also have an environment variable to disable the cache store for pipelineloop? I think the product team needs some time to migrate this behavior, so is better to have some flag that can turn it on/off.